launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/data/LoginStatus.java

```diff
@@ -14,13 +14,26 @@
 
 package com.google.launchpod.data;
 
+import java.util.ArrayList;
+
 /** An item on a comment list. */
 public final class LoginStatus {
   public final boolean isLoggedIn;
   public final String message;
+  public final ArrayList<UserFeed> feeds;
+
+  public static final String USER_FEED_KEY = "UserFeed";
+  public static final String TITLE_KEY = "title";
+  public static final String DESCRIPTION_KEY = "description";
+  public static final String NAME_KEY = "name";
+  public static final String EMAIL_KEY = "email";
+  public static final String XML_STRING_KEY = "timestamp";
```

Comment:
i think you want a different key for this

Reply:
done.